internal/rest/dequeue_nodes.go

```diff
@@ -415,7 +416,11 @@ func (rest *RestOperations) ExecuteRestAndPopulateCache(rest_ops []*utils.RestOp
 									utils.AviLog.Infof("key: %s, msg: Error is not of type AviError, err: %v, %T", key, rest_ops[i].Err, rest_ops[i].Err)
 									continue
 								}
-								retryable, fastRetryable := rest.RefreshCacheForRetryLayer(publishKey, aviObjKey, rest_ops[i], aviError, aviclient, avimodel, key)
+								retryable, fastRetryable, pushWithMessage := rest.RefreshCacheForRetryLayer(publishKey, aviObjKey, rest_ops[i], aviError, aviclient, avimodel, key)
+								if pushWithMessage {
```

Comment:
Why are we populating the cache on `this` error where the IP address request couldn't be fulfilled?

Isn't it much simpler to just handle this error as a special case outside of the RefreshCacheRetryLayer method? Since technically you aren't going to retry anyway?

@monotosh-avi pls review this code.

Reply:
Yes I think that would be better. 
If the error is of this special type, then we can handle it before line no : 409, and continue to the next error. 
Also, what happens if we need to retry for another error in the same error ? We would again see this special vsvip error  .. but I guess there is no way to avoid that.